index.yaml

```diff
@@ -20945,9 +20955,9 @@ entries:
     version: 1.2.1
   - apiVersion: v2
     appVersion: 1.2.0
-    created: "2024-09-05T13:21:20.070471945Z"
+    created: "2024-12-13T13:21:53.111318705Z"
     description: Helm chart for Tetragon
-    digest: 711faa2847966e52323f92e50ca5da69a462046470a2427176d273b1e1899d8e
+    digest: c3bfb7cebf8fd777120c07ea1b6707436e6219fdde4bd9966f55bfbc79a5c295
```

Comment:
Why are these modified?

Reply:
No idea 🤷🏼 

The commit is automatically generated from a script (`./generate_helm_release.sh`)

Reply:
Yes... OK let's assume the script works, I guess you'll see soon enough if someone complains :) :+1:

Reply:
And the checksum seems wrong 🤔 

```
sha256sum tetragon-1.2.0.tgz tetragon-1.3.0.tgz 
711faa2847966e52323f92e50ca5da69a462046470a2427176d273b1e1899d8e  tetragon-1.2.0.tgz
19bb9f3777ec823d43e7e5a1d2d6c8ef2320b54372f493d3fe460a0f842995e3  tetragon-1.3.0.tgz
```